modules/core/04-channel/keeper/handshake_test.go

```diff
@@ -220,22 +188,23 @@ func (suite *KeeperTestSuite) TestChanOpenTry() {
 			})
 
 			suite.chainB.CreatePortCapability(suite.chainB.GetSimApp().ScopedIBCMockKeeper, ibctesting.MockPort)
-			portCap = suite.chainB.GetPortCapability(ibctesting.MockPort)
 		}, false},
-		{"connection does not support ORDERED channels", func() {
-			path.SetupConnections()
-			path.SetChannelOrdered()
-			err := path.EndpointA.ChanOpenInit()
-			suite.Require().NoError(err)
 
-			// modify connA versions to only support UNORDERED channels
-			path.EndpointA.UpdateConnection(func(c *connectiontypes.ConnectionEnd) {
-				c.Versions = []*connectiontypes.Version{connectiontypes.NewVersion("1", []string{"ORDER_UNORDERED"})}
-			})
+		// This test doesn't check the correct error path. This is the actual error we get:
```

Comment:
Can we uncomment the test and try again? It looks like the reported error message should be different now that `AuthenticateCapability` calls were removed from `ChanOpenInit` and `ChanOpenTry` handshakes

Reply:
ah sorry! This was poorly phrased on my side!

Basically what happens is:

* On main, we get the error `caller does not own port capability for port ID mock: invalid port` (and thus the test passes, because `expPass == false`
* On this PR, since we removed the check for capability, we get `err == nil` and thus the test fails.

Reply:
It turns out it was a switcheroo of `A` and `B` chains, we were updating the wrong connection. Another reason for expected error tests!